generative_models/VanillaGAN_PyTorch/eval.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+import torchvision.transforms as transforms
+import torchvision
+import matplotlib.pyplot as plt
+import imageio
+import os
+
```

Comment:
stray lines

Reply:
no more than a maximum of one line has been allotted at meaningful places :)